pkg/datastore/datastore_with_txcontext_test.go

```diff
@@ -189,7 +191,7 @@ func TestSingleTxCrud(t *testing.T) {
 		queryResult := &AppUser{Id: record.Id}
 		err = ds.Find(txCtx, queryResult)
 		assert.NoError(err)
-		assert.Equal(record, queryResult)
+		cmp.Equal(record, &queryResult, cmpopts.IgnoreFields(AppUser{}, "CreatedAt", "UpdatedAt", "DeletedAt"))
```

Comment:
nit: can be taken up later, helper method that prints the records too? Does it assert?